### PR TITLE
No attr rel defaults

### PR DESCRIPTION
**Pull request by gvdent**
Makes noRelations and noAttributes default for query so the default uses the new template

### CODE REVIEW

File: test/WeaverQuery.test.coffee [hunk 1]
```diff
@@ -187,7 +188,7 @@ describe 'WeaverQuery Test', ->
 
     a.save().then(->
       new Weaver.Query()
-      .noRelations()
+      .noRelationNodes()
```
Comment (mingushead) [inline test/WeaverQuery.test.coffee]: This is unnecessary, right? Should I add one more duplicate test without this line to make sure the defaults stay set up correctly?

File: src/WeaverQuery.coffee [hunk 2]
```diff
@@ -209,5 +222,10 @@ class WeaverQuery
   nativeQuery: (query)->
     Weaver.getCoreManager().nativeQuery(query, Weaver.getInstance().currentProject().id())
 
+  includeAttributesAndRelations: ->
```
Comment (malamili) [inline src/WeaverQuery.coffee]: what does this mean? How is it different from noRelations and noAttributes?

**Comment by malamili** [comment]
Whats the purpose of ```includeAttributesAndRelations```?

File: src/WeaverQuery.coffee (hunk 2 above)
Comment (gvdent) [inline src/WeaverQuery.coffee]: Ugh, this bothers me as well. The options differ right now in the following manner:

includeAttributesAndRelations allows loading attribute and relation nodes directly. If this option is skipped loading a node with an id that is not an individual will fail (this is new template optimization behaviour).

As far as I understand noRelation and noAttributes determine whether the attribute/relations "children" nodes of actual queried nodes are loaded. So, querying for the 'individual-1' node will also load the 'individual-1-relation-1', 'individual-1-attribute-1' and 'individual-1-attribute-2' nodes. Correct me if I'm wrong on the expected behaviour but that I how I understood it from the tests.

I'm still pondering what to do with the options: whether to merge them or rename them in a way that their intention is more obvious.

Comment (malamili) [inline src/WeaverQuery.coffee]: Oeps! Haha you understood it wrong..noRelation and noAttributes mean that these nodes dont get returned as root nodes (so not meaning the children of the individuals)..

Comment (malamili) [inline src/WeaverQuery.coffee]: I think noRelation and noAttributes is what `NOT includeAttributesAndRelations` means right now

**Pull request merged**